File: 3ti/3tip_php/cw3_ob1/Student.php
<?php

class Student
{
    // private int $a;
    // private string $name;

    public function __construct(  public int $a, private string $name ) {
    }
    // public function __construct(int $a, string $name)
    // {
    //     $this->a = $a;
    //     $this->name = $name;
    // }
    public function __toString(): string
    {
        return "<p>a = {$this->a} name = {$this->name}</p>";
    }
   
}

File: 3ti/web/mvc_ef_mssql/mvc_ef_mssql/Models/Student.cs
using System.ComponentModel.DataAnnotations;

namespace mvc_ef_mssql.Models
{
    public class Student
    {
        [Key]
        public int Id { get; set; }

        [Required(ErrorMessage = "Podaj imię")]
        [Display(Name = "Imię")]
        [MinLength(2,ErrorMessage = "Za krótkie")]
        public string FirstName { get; set; }


        [Required(ErrorMessage = "Podaj nazwisko")]
        [Display(Name = "Nazwisko")]
        [MinLength(2, ErrorMessage = "Za krótkie")]
        public string LastName { get; set; }

        [Required(ErrorMessage = "podaj wiek")]
        [Range(1,150,ErrorMessage = "wiek z przedziału (1,150)")]
        public int? Age { get; set; }
    }
}